.vale/styles/OpenShiftAsciiDoc/SuggestAttribute.yml

```diff
@@ -13,7 +13,25 @@ swap:
   (?<!Red Hat OpenShift )distributed tracing data collection: "{OTELShortName}"
   Alibaba Cloud: "{alibaba}"
   Bare Metal Event Relay: "{redfish-operator}"
+  CAPA: "{capa-first} or {capa-short}"
```

Comment:
Use | instead of or

Reply:
I'm leaving this i your capable hands, @aireilly

Reply:
fancy! thanks :nerd_face: